tests/unittests/Foundation/NSURLTests.m

```diff
@@ -225,16 +240,20 @@ void testNSURLMethod(SEL selector, NSURL* input, id argument, NSURL* expected) {
 
     // change current dir to app start up path
     ASSERT_TRUE(SetCurrentDirectoryW(startUpPath) != 0);
+    NSURL* baseURL = [NSURL fileURLWithPath:[manager currentDirectoryPath]];
+    NSURL* targetURL = [NSURL URLWithString:@"data/NSFileManagerUT.txt" relativeToURL:baseURL];
 #else
     char tempBuffer[_MAX_PATH];
     wcstombs(tempBuffer, startUpPath, _MAX_PATH);
 
     ASSERT_TRUE(chdir(dirname(tempBuffer)) == 0);
+    NSURL* baseURL = [NSURL fileURLWithPath:[manager currentDirectoryPath]];
+    NSURL* targetURL = [NSURL URLWithString:@"OSX/data/NSFileManagerUT.txt" relativeToURL:baseURL];
```

Comment:
worth noting also that this is currently not failing on my machine... unless for some reason in a future release of Foundation they made currentDirectoryPath return the dir one up? that seems really unlikely though...